File: Frontend/Frontend/src/app/_dtos/updateTaskItemDto.ts
import { WorkflowItem } from "../_models/workflowitemmodel"

export class UpdateTaskItemDto {
    public id: string = ''
    public title: string = ''
    public details: string = ''
    public priority: string = '1'
    public deadline: string = ''
    public assigneeId: string = ""
    public workflowItems: Array<WorkflowItem> = []
}

File: Backend/Backend/DTOs/UpdateTaskItemDto.cs
using Backend.Models;

namespace Backend.DTOs
{
    public class UpdateTaskItemDto
    {
        public string Id { get; set; }
        public string Title { get; set; }
        public string Details { get; set; }
        public string Priority { get; set; }
        public DateTime Deadline { get; set; }
        public string AssigneeId { get; set; }
        public ICollection<WorkflowItem> WorkflowItems { get; set; }
    }
}
